app/assets/javascripts/application.js

```diff
@@ -1,4 +1,4 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
+// This is a manifest file that'll be compiled into et_dropzone_uploader.js, which will include all the files
```

Comment:
Why will this manifest get compiled into et_dropzone_uploader when its in app/assets/javascripts ?

Reply:
Good point - fixed in b055fec thank you!